e2e-pw/src/oss/specs/smoke-tests/saved-views.spec.ts

```diff
@@ -0,0 +1,320 @@
+import { test as base, expect } from "src/oss/fixtures";
+import { SavedViewsPom } from "src/oss/poms/saved-views";
+import { getUniqueDatasetNameWithPrefix } from "src/oss/utils";
+
+export const ColorList = [
+  "Gray",
+  "Blue",
+  "Purple",
+  "Red",
+  "Yellow",
+  "Green",
+  "Pink",
+  "Orange",
+  "Purple",
+];
+
+export type Color =
+  | "Gray"
+  | "Blue"
+  | "Purple"
+  | "Red"
+  | "Yellow"
+  | "Green"
+  | "Pink"
+  | "Orange"
+  | "Purple";
+
+export const updatedView = {
+  name: "test updated",
+  description: "test updated",
+  color: "Yellow" as Color,
+};
+
+export const updatedView2 = {
+  name: "test updated 2",
+  description: "test updated 2",
+  color: "Orange" as Color,
+  slug: "test-updated-2",
+};
+
+const testView = {
+  id: 0,
+  name: "test",
+  description: "description",
+  color: "Gray" as Color,
+  newColor: "Blue" as Color,
+  slug: "test",
+};
+
+const testView1 = {
+  id: 1,
+  name: "test 1",
+  description: "description ",
+  color: "Gray",
+  newColor: "Orange",
+  slug: "test-1",
+};
+
+const testView2 = {
+  id: 2,
+  name: "test 2",
+  description: "description 2",
+  color: "Gray",
+  newColor: "Yellow",
+  slug: "test-2",
+};
+
+const datasetName = getUniqueDatasetNameWithPrefix("smoke-quickstart");
+
+const test = base.extend<{ savedViews: SavedViewsPom }>({
+  savedViews: async ({ page }, use) => {
+    await use(new SavedViewsPom(page));
+  },
+});
+
+test.describe("saved views", () => {
+  test.beforeAll(async ({ fiftyoneLoader }) => {
+    await fiftyoneLoader.loadZooDataset("quickstart", datasetName, {
+      max_samples: 5,
+    });
+  });
+
+  test.beforeEach(async ({ page, fiftyoneLoader, savedViews }) => {
+    await fiftyoneLoader.waitUntilLoad(page, datasetName);
+    await delete_saved_view(savedViews, testView.name);
+    await delete_saved_view(savedViews, updatedView2.slug);
+  });
+
+  async function delete_saved_view(savedViews, slug: string) {
```

Comment:
- change to camelCase `deleteSavedView`
- nit: but might be a good idea to hoist this function outside of the test scope, somewhere below line 68